modules/AM_ProjectTemplates/metadata/studio.php

```diff
@@ -1,8 +1,9 @@
 <?php
-/*********************************************************************************
+/**
+ *
  * SugarCRM Community Edition is a customer relationship management program developed by
  * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
- *
+ * *
```

Comment:
Is this the wrong License?

Reply:
Those are errors... from files with a different license notice that I've not founded!
Will try to fix